README.md

```diff
@@ -1,2 +1,44 @@
 # docker-liberica-jre
-JRE Docker images using Alpine, Liberica and glibc
+
+## Introduction
+
+Provides base Java Docker images using Alpine, [Liberica] an glibc.
+
+Builds run daily, but updates to images are only pushed if Apline or the JVM is updated.
+
+This repository is derived from the images defined in the
+[Liberica public repo](https://github.com/bell-sw/Liberica) licenced under GPL 2.0.
+
+## Images
+
+This repo currently maintains builds for versions 11 and 17 of the JRE. The image name/tags are as follows:
+
+```
+ghcr.io/permutive-engineering/permutive-jre-liberica:17
+```
+
+```
+ghcr.io/permutive-engineering/permutive-jre-liberica:11
+```
```

Comment:
I did wonder if the `permutive-*` prefix here is redundant btw? Potentially too late as we are using it, but we do have the organisation there already.

On the other hand it makes it much easier to work out the image if all you can see is everything after the last `/`...

Reply:
> On the other hand it makes it much easier to work out the image if all you can see is everything after the last `/`...

Yes, that's why I wanted to leave it. While most of the time it shows the full image name we can't control that it will be the case in the future.